frontend/pages/admin/UserManagementPage/helpers/userManagementHelpers.ts

```diff
@@ -0,0 +1,54 @@
+import { isEqual } from "lodash";
+
+import { IInvite } from "interfaces/invite";
+import { IUser, IUserUpdateBody } from "interfaces/user";
+import { IFormData } from "../components/UserForm/UserForm";
+
+type ICurrentUserData = Pick<
+  IUser,
+  "global_role" | "teams" | "name" | "email" | "sso_enabled"
+>;
+
+/**
+ * Helper function that will compare the current user with data from the editing
+ * form and return an object with the difference between the two. This can be
+ * be used for PATCH updates when updating a user.
+ * @param currentUserData
+ * @param formData
+ */
+const generateUpdateData = (
```

Comment:
Could be nice to see a test for this function.

Reply:
will add